JimRoyal-montreal.md

```diff
@@ -9,5 +9,9 @@
 
 ## Things to Do
 
+- Cycle the Lachine Canal
+- Visit the MMFA
```

Comment:
@chanlon27 The MMFA Is the Montreal Museum of Fine Arts. 

@sathya-ghub It was 90F yesterday. Warm enough? 😄